### ISSUE TITLE
Implement Tiny Neural Net for Busy/Calm Detection (Option 2)

### ISSUE THREAD

**Issue by PianoPlayer09** [PianoPlayer09/EnviroApp#1]
Implement a tiny neural network solution for detecting busy vs calm environments using audio features.

- Extract 20–40 MFCCs from 1–2 sec of audio clips
- Build a simple 1D CNN or shallow MLP (example: 2-3 dense layers with ReLU, output sigmoid)
- Train using a small dataset: record/collect 30–100 short “calm” and “busy” audio clips
- Train the model to output 0 (calm) or 1 (busy)
- Convert the trained model to OpenVINO IR for efficient CPU inference
- Document training and conversion steps
- Test the OpenVINO model on sample clips

Reference architecture:
```python
model = tf.keras.Sequential([
    tf.keras.layers.Input(shape=(40,)),  # MFCCs
    tf.keras.layers.Dense(32, activation='relu'),
    tf.keras.layers.Dense(16, activation='relu'),
    tf.keras.layers.Dense(1, activation='sigmoid')
])
```
